### DISCUSSION TITLE
Problems with creating from png file

### DISCUSSION THREAD

**Discussion by johnho31** [arkypita/LaserGRBL#1190] [Q&A]
I have had laserGRBL with my Laser engraver and have used it several times. All i've done is open a png click next set the size and power and click create - this has worked on 4 or 5 projects with no isssue and I was very happy, but now when I click create I just get a few lines. I even tried increasing the number of lines per mm to the maximm of 20 but although the rendered image does show what it is it doesnt look anywhere as crisp as it used to..

Can someone advise what might have gone wrong here - I'm not experienced with this but as I say I've used it several times before with no issues.

thanks in advance
John

**Comment by gmmanonymus111** [comment]
I would suggest take a look at brightness, contrast and white clip sliders. Sometimes they need to be changed to produce better quality images
![image](https://user-images.githubusercontent.com/9658243/101369725-ed30ed80-38a8-11eb-904b-55e38e95c055.png)
![image](https://user-images.githubusercontent.com/9658243/101369954-2f5a2f00-38a9-11eb-93a8-56e42d8e7d68.png)
